### PR TITLE
fix: the row height is adjusted incorrectly when the column are added dynamically

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* related issue(#1332)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Review by adhrinae — APPROVED** [review]
리뷰완료합니다

File: packages/toast-ui.grid/src/dispatch/renderState.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function setCellHeight(
   cellHeightMap[rowIndex][columnName] = Math.max(height, defaultRowHeight);
 }
 
-export function removeCellHeight({ renderState }: Store, rowIndex: number) {
+export function removeRowHeight({ renderState }: Store, rowIndex: number) {
   const { cellHeightMap } = renderState;
   delete cellHeightMap[rowIndex];
```
Comment (seonim-ryu) [inline packages/toast-ui.grid/src/dispatch/renderState.ts:41]: `delete renderState.cellHeightMap[rowIndex]` 이렇게 줄일수도 있겠다 생각했지만..
가독성이 몹시 떨어지는군요.. 🙄

**Review by seonim-ryu — APPROVED** [review]
이견 없습니다 고생하셨어요~ 👍

**Pull request merged**